ruby/rubocop-ruby.yml

```diff
@@ -262,20 +284,10 @@ Style/BlockDelimiters:
     - expect
   Exclude:
     - 'spec/factories.rb'
-
-Style/BracesAroundHashParameters:
```

Comment:
This was removed because ruby 2.7 introduces a distinction between keyword args and a hash parameter. See https://github.com/rubocop-hq/rubocop/pull/7643